xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -755,7 +756,30 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       // Unsupported type?
       xbmc_jnienv()->ExceptionDescribe();
       xbmc_jnienv()->ExceptionClear();
-      continue;
+
+      // There is some confusion for video/VC1 vs. video/wvc1 especially on Sony devices, while
+      // JANA defines VC-1 as video/vc1, Android API defines it as video/wvc1
+      bool isVC1 = StringUtils::Contains(m_codecname, "vc1", false);
```

Comment:
Shouldn't you look for "wvc1" (including leading `w` if you want to replace exactly this string?

Reply:
Nope - I parse the "decoder" names, which are always VC1 or vc1 or something and btw. it will also catch WVC1.

The format actually really is VC1 and not the Advancec VC1 known as WVC1. So VC1 check is fine I think.